common-consortia/src/main/resources/common-consortia/initData.feature

```diff
@@ -140,8 +140,7 @@ Feature: init data for consortia
 
   @PostPermissions
   Scenario: Post specified permissions to the user
-    * def consortiaPermissions = $consortiaPermissions[*].name
-    * def permissions = karate.get('extPermissions', consortiaPermissions)
+    * def permissions = karate.get('permissions', [])
```

Comment:
this module will be used by others as well, so please write some clarifcation, couldn't understand